tests/integration_tests/multi_bootstrap.py

```diff
@@ -14,10 +14,8 @@
 
 from __future__ import print_function
 
-import os
-import sys
+import traceback
```

Comment:
This test was heavily simplified. The same failures happen in roughly a third of the execution time.